src/main.cpp

```diff
@@ -23,6 +23,11 @@ void setup() {
 
   BLEDevice::init("AirPods 69");
 
+  // Increase the BLE Power to 9dBm (MAX)
+  esp_ble_tx_power_set(ESP_BLE_PWR_TYPE_DEFAULT, ESP_PWR_LVL_P9); 
+  esp_ble_tx_power_set(ESP_BLE_PWR_TYPE_ADV, ESP_PWR_LVL_P9);
+  esp_ble_tx_power_set(ESP_BLE_PWR_TYPE_SCAN ,ESP_PWR_LVL_P9);
```

Comment:
Only need to set the power for advertising.

```suggestion
  esp_ble_tx_power_set(ESP_BLE_PWR_TYPE_ADV, ESP_PWR_LVL_P9);
```